src/main/java/seedu/address/logic/Messages.java

```diff
@@ -41,16 +41,37 @@ public static String format(Person person) {
         final StringBuilder builder = new StringBuilder();
         builder.append(person.getName())
```

Comment:
I like that you have abstracted out the builder.append(person) and builder.append('some text') for email, address, tags and date of last visit into a single method respectively. Perhaps you might want to do the same for name and phone too, just to make it consistent.